src/commands/fun/activity.ts

```diff
@@ -0,0 +1,64 @@
+import { SlashCommandBuilder } from "@discordjs/builders";
+import { CommandInteraction, MessageActionRow, MessageButton, MessageEmbed, Permissions } from "discord.js";
+
+import { DiscordClient, SlashCommandHandler } from "../../client";
+
+const allApplications = {
+  youtube: "880218394199220334",
+  poker: "755827207812677713",
+  betrayal: "773336526917861400",
+  fishing: "814288819477020702",
+  chess: "832012774040141894",
+  lettertile: "879863686565621790",
+  wordsnack: "879863976006127627",
+  doodlecrew: "878067389634314250",
+  awkword: "879863881349087252",
+  spellcast: "852509694341283871",
+};
+
+export default class ActivityCommand extends SlashCommandHandler {
+  builder = new SlashCommandBuilder()
+    .setName("activity")
+    .setDescription("Generate discord together activity invite")
+    .addStringOption((option) => option.setName("name").setDescription("The activity name").addChoices([["Watch Together", "youtube"], ["Poker Night", "poker"], ["Betrayal.io", "betrayal"], ["Chess In The Park", "chess"], ["Fishington.io", "fishing"], ["Poker Night", "poker"], ["Lettertile", "lettertile"], ["Wordsnack", "wordsnack"], ["Doodlecrew", "doodlecrew"], ["Awkword", "awkword"], ["Spellcast", "spellcast"]]).setRequired(true))
+    .addChannelOption((option) => option.setName(channel").setDescription("The voice channel").setRequired(false));
+
+  async handle(interaction: CommandInteraction, _client: DiscordClient): Promise<void> {
+    const name = interaction.options.getString("name", true);
+    const channel = interaction.options.getChannel("channel", false) ?? interaction.member?.voice?.channel;
+
+    const applicationId = allApplications[name];
```

Comment:
TypeScript also doesn't allow for indexing into an object by itself, but you can make the object into a `Record`, which should work:
```ts
const allApplications: Record<string, string> = {
...
};
```